src/rest/check.php

```diff
@@ -9,7 +9,7 @@
 $sidvalue = session_id();
 function getUserIP()
 {
-    if ($_SERVER['HTTP_X_FORWARDED_FOR']) {
+    if (isset($_SERVER['HTTP_X_FORWARDED_FOR'])) {
```

Comment:
On installations that have the HTTP_X_FORWARDED_FOR index this will always be successful. Please add `&& $_SERVER['HTTP_X_FORWARDED_FOR']` to the condition.